core/include/gnuradio-4.0/AtomicBitset.hpp

```diff
@@ -35,7 +35,11 @@ class AtomicBitset {
     requires(!isSizeDynamic)
     {
         for (auto& word : _bits) {
+#if defined(_WIN32)
+            word.store(0ULL, std::memory_order_relaxed);
```

Comment:
Correct portability bug identified (i.e. `std::size_t` has platform-specific backing types), but rather than using an if-def macro, we should use the `UZ` instead of the `UL` or `ULL` literal.

Reply:
*N.B. for info: the original code was written for C++17/20 and older gcc/clang versione where `UZ` wasn't available yet.*